### PR TITLE
Add: authenticator documentation in react md file

### PR DESCRIPTION
**Pull request by NikhilSejwal7**

### CODE REVIEW

File: getting-started/quickstart-guides/angular.md [hunk 1]
```diff
@@ -84,7 +84,8 @@ npm install --save imagekitio-angular
 Before the SDK can be used, let's learn about and obtain the requisite initialization parameters:
 
 * `urlEndpoint` is a required parameter. This can be obtained from the [URL-endpoint section](https://imagekit.io/dashboard/url-endpoints) or the [developer section](https://imagekit.io/dashboard/developer/api-keys) on your ImageKit dashboard.
-* `publicKey` and `authenticationEndpoint` parameters are optional and only needed if you want to use the SDK for client-side file upload. These can be obtained from the [developer section](https://imagekit.io/dashboard/developer/api-keys) on your ImageKit dashboard.
+* `publicKey` and `authenticator` parameters are optional and only needed if you want to use the SDK for client-side file upload. These can be obtained from the [developer section](https://imagekit.io/dashboard/developer/api-keys) on your ImageKit dashboard.
```
Comment (imagekitio) [inline getting-started/quickstart-guides/angular.md]: `authenticator` cannot be obtained from developer section. I think you replaced `authenticationEndpoint` with `authenticator` in all files. Same feedback for rest of this PR.

File: getting-started/quickstart-guides/angular.md [hunk 2]
```diff
@@ -678,9 +669,32 @@ For this, we will use the `ik-upload` component as well as a couple of event han
...
     (onError)="handleUploadError($event)"
     (onSuccess)="handleUploadSuccess($event)" 
-    >
-  </ik-upload>
+    [authenticator]="authenticator"
+></ik-upload>
+```
+
+In `app.component.js` file:
+
```
Comment (imagekitio) [inline getting-started/quickstart-guides/angular.md:677]: Add some useful information e.g. `you can call a secure endpoint on your backend and pass any custom request headers for authentication purpose`.

File: getting-started/quickstart-guides/vuejs.md [hunk 3]
```diff
@@ -365,8 +363,6 @@ export default {
 </script>
 ```
 
-> `authenticationEndpoint` should be implemented in your backend. The SDK makes an HTTP GET request to this endpoint and expects a JSON response with three fields i.e. `signature`, `token`, and `expire`. [Learn how to implement authenticationEndpoint](https://docs.imagekit.io/api-reference/upload-file-api/client-side-file-upload#how-to-implement-authenticationendpoint-endpoint) on your server.
-
 For this quickstart guide, we have provided a sample implementation of `http://localhost:3001/auth` in Node.js.
```
Comment (imagekitio) [inline getting-started/quickstart-guides/vuejs.md:368]: This line alone doesn't make much sense. With above paragraph removed, the whole context is lost.

**Review by imagekitio — CHANGES_REQUESTED** [review]

File: getting-started/quickstart-guides/angular.md (hunk 1 above)
Comment (NikhilSejwal7) [reply getting-started/quickstart-guides/angular.md]: Fixed

File: getting-started/quickstart-guides/angular.md (hunk 2 above)
Comment (NikhilSejwal7) [reply getting-started/quickstart-guides/angular.md:677]: Added text and example with custom request headers

File: getting-started/quickstart-guides/vuejs.md (hunk 3 above)
Comment (NikhilSejwal7) [reply getting-started/quickstart-guides/vuejs.md:368]: Added description above

**Pull request closed without merging**